wandb/util.py

```diff
@@ -99,19 +98,23 @@ def get_platform_name() -> str:
 
 
 # TODO(sentry): This code needs to be moved, sentry shouldn't be initialized as a
-# side effect of loading a module.
+#  side effect of loading a module.
```

Comment:
in the current design if we want to disable sdk_sentry we still need to provide the env variable before the import correct?

Reply:
yup